### ISSUE TITLE
Still waiting for the source code update

### ISSUE THREAD

**Issue by iherbak** [makerbase-mks/MKS-DLC32-FIRMWARE#38]
Somebody please do it, it is almost Christmas time. Give us this present.
Thanks

**Comment by OriginaldCrypt** [comment]
Published source code is just unusable. It's a pity that this board+display, potentially perfect for lots of personal projects, cannot be used because of the firmware.

**Comment by Tashadra** [comment]
I fully agree, also getting generic answers to questions when sending emails to makerbase (that is if the emails don't bounce back)

**Comment by FrankHovis** [comment]
It won't happen, just give up and move on to something that isn't Makerbase as they seem to do the same with all their products.

**Comment by dosAnjosGustavo** [comment]
please guys, share the latest source code with the community @makerbase-mks @solawc @elf128 @abinoer @mks-viva @MS1987 @nehilo @KevinOConnor
